### PR TITLE
Disable mouse wheel hover canvas

### PR DESCRIPTION
**Pull request by ismaeldcom**
#16 I know there is another PR for this, but I think that is better to disable the wheel event.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ function appendCanvasesToHTML() {
 
 document.getElementById('file-input').onchange = function (event) {
     const fr = new FileReader();
-    fr.onload = setupNifti;
+    fr.onload = (event) => setupNifti(event.target.result);
```
Comment (Irev-Dev) [inline script.js]: Had to fix this, since you change the setupNifti to take a file instead of the fileReader event

Comment (ismaeldcom) [reply script.js]: Oh! I push this commit for #8 into master instead of the remote-data branch by mistake, sorry and thanks for catching that bug.

**Comment by ismaeldcom** [comment]
@Irev-Dev I really enjoy contributing to projects like this, is very different from what I work every day.

File: script.js (hunk 1 above)
Comment (Irev-Dev) [reply script.js]: Yeah I was surprise to see those changes bundled in this pr. All good 👌

**Comment by Irev-Dev** [comment]
@ismaeldcom Good to hear, and that's true for me to, different from work. 
Since this project is starting to look really nice I should make an effort to promote it a little, see If I can get some sciency folk interested in using it somehow, maybe integrate into an existing website. I have a few people I know with that background who might have some ideas.

(p.s. if you want to contribute any more you should pull master, I just set up eslint and there is heaps of white space changes, conflict galore. I should have set up linting ages ago.)

**Pull request merged**